src/Storage/Storage/Blob/Cmdlet/CopyAzureStorageBlob.cs

```diff
@@ -322,40 +321,68 @@ private void CopyBlobSync(IStorageBlobManagement destChannel, string srcUri, str
         private async Task CopyFromUri(long taskId, IStorageBlobManagement destChannel, Uri srcUri, BlobBaseClient destBlob)
         {
             bool destExist = true;
-            Track2Models.BlobType? destBlobType = Util.GetBlobType(destBlob);
+            //Track2Models.BlobType? destBlobType = Util.GetBlobType(destBlob);
+            Track2Models.BlobType? srcBlobType = Util.GetBlobType(new BlobBaseClient(srcUri, ClientOptions), true).Value;
+            Track2Models.BlobType? destBlobType = Util.GetBlobType(new BlobBaseClient(srcUri, ClientOptions), true).Value;
```

Comment:
Is srcUri correct here for destBlobType?

Reply:
Yes. The logic here is to initialize destBlobType to be equal to the source blob type first as the default value, and then change it later in the code if necessary.